script.js

```diff
@@ -212,6 +219,12 @@ window.addEventListener('load', () => {
       projectModal.style.top = `${window.pageYOffset}px`;
     });
   }
+
+  // Insert Form data from formData object into contact form
+  formData = JSON.parse(localStorage.getItem('formData'));
+  for (let i = 0; i < inputs.length; i += 1) {
+    inputs[i].value = formData[inputs[i].name];
+  }
```

Comment:
- [ ] Your form data is not being submitted to local storage there is a problem with your functions, Please make sure the local storage functionality works fine.

### See the image of the error.

![image](https://user-images.githubusercontent.com/83803414/162301085-47b4eaa8-0054-4319-b983-8f4bb21a7780.png)

Reply:
Oh yeah 😊 I made a mistake when I was trying to fix the linter error.